packages/slice-machine/lib/env/manifest.ts

```diff
@@ -42,43 +42,51 @@ export function extractRepo(parsedRepo: ParseResult): string | undefined {
 }
 
 function handleManifest(cwd: string, validate = false): ManifestInfo {
-  const maybeManifest = retrieveManifest(cwd, validate);
-  if (maybeManifest.exists === false) {
-    return {
-      state: ManifestState.NotFound,
-      message: Messages[ManifestState.NotFound],
-      content: null,
-    };
-  }
+  try {
+    const maybeManifest = retrieveManifest(cwd, validate);
+    if (maybeManifest.exists === false) {
+      return {
+        state: ManifestState.NotFound,
+        message: Messages[ManifestState.NotFound],
+        content: null,
+      };
+    }
+
+    if (maybeManifest.errors) {
+      const messages = formatValidationErrors(maybeManifest.errors, {});
+      const message = messages.map((error) => "[sm.json] " + error).join("\n");
+
+      return {
+        state: ManifestState.InvalidJson,
+        message,
+        content: null,
+      };
+    }
 
-  if (maybeManifest.errors) {
-    const messages = formatValidationErrors(maybeManifest.errors, {});
-    const message = messages.map((error) => "[sm.json] " + error).join("\n");
+    if (maybeManifest.content === null) {
```

Comment:
This can't actually happen right ? Since we're catching it.

Reply:
It's unlikely but in the typescript thinks it can